pkg/schedule/schedulers/evict_slow_trend.go

```diff
@@ -40,7 +40,7 @@ import (
 const (
 	alterEpsilon               = 1e-9
 	minReCheckDurationGap      = 120 // default gap for re-check the slow node, unit: s
-	defaultRecoveryDurationGap = 600 // default gap for recovery, unit: s.
+	defaultRecoveryDurationGap = 900 // default gap for recovery, unit: s.
```

Comment:
Let's set it to 1800. It should be enough time for TiKV disk failover.

Reply:
ok